codespan-reporting/src/term/views.rs

```diff
@@ -454,6 +454,7 @@ where
 /// Output a short diagnostic, with a line number, severity, and message.
 pub struct ShortDiagnostic<'diagnostic, FileId> {
     diagnostic: &'diagnostic Diagnostic<FileId>,
+    show_notes: bool,
```

Comment:
On a later read of this I'm not really happy with the coupling of these two views, but I think that can be fixed in a later PR. Let's merge this - it's been open for too long!